plan/plan.go

```diff
@@ -1254,7 +1260,16 @@ func resolveComponentCommon(commons ...v2.Common) ComponentCommon {
 		}
 	}
 
+	extraTemplates := []ExtraTemplate{}
+	for _, v := range v2.ResolveExtraTemplates(commons...) {
+		extraTemplates = append(extraTemplates, ExtraTemplate{
+			Files:     v.Files,
+			Overwrite: v.Overwrite,
+		})
+	}
```

Comment:
Potential opportunity to make a helper to consolidate same mapping functionality (2)